PDK_Abstraction/FinFET14nm_Mock_PDK/FinFET_Mock_PDK_Abstraction_v1.json

```diff
@@ -0,0 +1,436 @@
+{
+"_comment":"The rules defined in this file are discussed in FinFET_MOCK_PDK.ppt",
+
+"Abstraction": [
+
+{
+"Layer": "Poly",
+"LayerNo": 1,
+"Direction": "V",
+"Width": [14],
+"Space": [66],
+"MinL": [60],
+"MaxL": ["NA"],
+"End-to-End": ["NA"],
+"Offset": [0]
+},
+
+{
+"Layer": "Fin",
+"LayerNo": 3,
+"Direction": "H",
+"Width": [10],
+"Space": [32],
+"MinL": [100],
+"MaxL": ["NA"],
+"End-to-End": ["NA"],
+"Offset": [0]
+},
+
+{
+"Layer": "M1",
+"LayerNo": 6,
+"Direction": "V",
+"Color": 2,
+"Width": [32],
+"Space": [48],
+"MinL": [180],
+"MaxL": ["NA"],
+"End-to-End": [48],
+"Offset": [40]
```

Comment:
Offset appears to be new to this version. Can you expand upon why this is needed? Its probably to deal with alignment issues but I just want to make sure we are all on the same page.

Reply:
yes, this is to deal with alignment

Reply:
Thank you !